cognite/transformations_cli/commands/jobs.py

```diff
@@ -16,20 +16,41 @@
 @click.option("--id", help="List jobs by transformation id. Either this or --external-id must be specified.")
 @click.option("--external-id", help="List jobs by transformation external_id. Either this or --id must be specified.")
 @click.option("--limit", default=10, help="Limit for the job history, defaults to 10. Use -1 to retrieve all results.")
+@click.option("-i", "--interactive", is_flag=True, help="Display only 10 jobs at a time, paging through them.")
 @click.pass_obj
-def jobs(obj: Dict, id: Optional[int], external_id: Optional[str], limit: int = 10) -> None:
+def jobs(obj: Dict, id: Optional[int], external_id: Optional[str], limit: int = 10, interactive: bool = False) -> None:
     _, exp_client = get_clients(obj)
     is_id_exclusive(id, external_id)
     try:
+        id_str = None
         if id:
-            click.echo(f"Listing the latest jobs for transformation with id {id}:")
+            id_str = f"id {id}"
         if external_id:
             id = get_id_from_external_id(exp_client=exp_client, external_id=external_id)
-            click.echo(f"Listing the latest jobs for transformation with external_id {external_id}:")
+            id_str = f"id {external_id}"
+
+        if id_str:
+            click.echo(f"Listing the latest jobs for transformation with {id_str}:")
+        else:
+            click.echo("Listing the latest jobs for all transformations:")
+
         jobs = exp_client.transformations.jobs.list(limit=limit, transformation_id=id)
+
         if jobs:
-            click.echo("Resulting jobs:")
-            click.echo(print_jobs(jobs))
+            if interactive:
```

Comment:
Yeah, so this is due to the kind of half-baked way I did it. Turns out picking up standard-in directly in python is really hard to do in a general way. I just use "input" here, so you need to press enter after typing q, i.e. enter q completely.

It's ugly, but the alternative is complicated, and will involve some more packages, which I wanted to avoid. There is also no guarantee it will work everywhere.

Reply:
Essentially, this uses input, which will work over ssh, on all OSs, etc. The alternative would use some keyboard magic, which may be OS specific, and might not always work.